claims_test.go

```diff
@@ -1,39 +1,55 @@
 package jwtauth
 
 import (
+	"encoding/json"
 	"testing"
 
 	"github.com/go-test/deep"
 )
 
 func TestGetClaim(t *testing.T) {
-	tests := []struct {
-		allClaims map[string]interface{}
-		claim     string
-		expected  interface{}
-	}{
-		{nil, "data", nil},
-		{map[string]interface{}{"data": "foo"}, "data", "foo"},
-		{map[string]interface{}{"data": "foo"}, "data2", nil},
-		{map[string]interface{}{"data": "foo"}, "/data", "foo"},
+	data := `
 		{
-			map[string]interface{}{"data": map[string]interface{}{
-				"foo": "bar",
-			}}, "/data/foo", "bar",
-		},
-		{
-			map[string]interface{}{"data": map[string]interface{}{
-				"foo": "bar",
-			}}, "/data/foo2", nil,
-		},
+		  "a": 42,
+		  "b": "bar",
+		  "c": {
+		    "d": 95,
+		    "e": [
+		      "dog",
+		  	  "cat",
+		  	  "bird"
+		    ],
+			"f": {
+			  "g": "zebra"
+			}
+		  }
```

Comment:
Between the JSON linter and gofmt, the JSON here got a little funky! :-) Not sure if it's worth the effort but if it's easy to get this pretty, it would be nice.